hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -53,11 +53,12 @@ def _default_date_end(self):
 
     def _default_employee(self):
         company = self.env['res.company']._company_default_get()
-        employee = self.env['hr.employee'].search([
+        employees = self.env['hr.employee'].search([
             ('user_id', '=', self.env.uid),
-            ('company_id', '=', company.id),
-        ], limit=1)
-        return employee
+        ])
+        return employees.filtered(
+            lambda e: e.company_id == company
+        ) or employees.filtered(lambda e: not e.company_id)
```

Comment:
rework as you like this method, but it should return at most one record. Remember that for each user, you can have more than one employee.

In fact, I would go with:

```python
        employee = self.env['hr.employee'].search([
            ('user_id', '=', self.env.uid),
            ('company_id', 'in', [company.id, False]),
        ], limit=1, order="company_id ASC")
        return employee
```

Reply:
Yes, it's also more readable. Thanks!